ci-operator/config/red-hat-data-services/ods-ci/red-hat-data-services-ods-ci-releases-2.17__rhoai-ocp4.19-lp-interop.yaml

```
@@ -67,7 +71,7 @@ tests:
     - ref: rhoai-tests
     workflow: firewatch-ipi-aws
 zz_generated_metadata:
-  branch: release-2.16
+  branch: releases-2.17
   org: red-hat-data-services
   repo: ods-ci
-  variant: rhoai-ocp4.18-lp-interop
+  variant: rhoai-ocp4.19-lp-interop
```

```suggestion
  variant: rhoai-ocp4.19-upgrade
```

Or some other option that specifies upgrade testing and removes the use of `lp`.
Should also be reflected in config file name.